packages/config/tailwind-bases.js

```diff
@@ -0,0 +1,49 @@
+const plugin = require("tailwindcss/plugin");
+
+module.exports = plugin(function ({ addBase, theme, addComponents }) {
+  const h1Base = {
+    fontSize: "28px",
+    fontWeight: theme("fontWeight.semibold"),
+  };
+
+  const h2Base = {
+    fontSize: theme("fontSize.2xl"),
```

Comment:
Maybe it would be a good idea if we'd have a font size called `h1`, `h2`, etc, because that way we can easily set `font-h1` to override an h2; `<h2 className="font-h1">`. Also: If you make the font definition an array, the second value will be the leading 🔥